config/source/300-rabbitmqsource.yaml

```diff
@@ -1,51 +1,290 @@
-# Copyright 2020 The Knative Authors
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
 
+---
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  labels:
-    rabbitmq.eventing.knative.dev/release: devel
-    eventing.knative.dev/source: "true"
-    duck.knative.dev/source: "true"
-    knative.dev/crd-install: "true"
   annotations:
-    registry.knative.dev/eventTypes: |
-      [
-        { "type": "dev.knative.rabbitmq.event" }
-      ]
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
   name: rabbitmqsources.sources.knative.dev
 spec:
   group: sources.knative.dev
+  names:
+    categories:
+    - all
+    - knative
+    - eventing
+    - sources
+    - importers
+    kind: RabbitmqSource
+    listKind: RabbitmqSourceList
+    plural: rabbitmqsources
+    singular: rabbitmqsource
```

Comment:
@ChunyiLyu can we define a `shortName` for this resources?

Reply:
Sure we can. Any suggestions?

Reply:
Add short name `rmqsource` per suggestion from team slack: https://vmware.slack.com/archives/C02BPEY4FU2/p1646826542641919